gradle/libs.versions.toml

```diff
@@ -169,6 +170,7 @@ kotlinx-serialization-json = { module = "org.jetbrains.kotlinx:kotlinx-serializa
 ktor = { module = "io.ktor:ktor-client-android", version.ref = "ktor" }
 leakCanary = { module = "com.squareup.leakcanary:leakcanary-android", version.ref = "leakCanary" }
 moneta = { module = "org.javamoney.moneta:moneta-core", version.ref = "moneta" }
+navigationRecentsUrlSharing = { module = "com.stylianosgakis.navigation.recents.url.sharing:navigation-recents-url-sharing", version.ref = "navigationRecentsUrlSharing" }
```

Comment:
where is the lib to take a look at?:)

Reply:
https://github.com/StylianosGakis/navigation-recents-url-sharing :D

The logic is here https://github.com/StylianosGakis/navigation-recents-url-sharing/blob/main/library/navigation-recents-url-sharing/src/main/java/com/stylianosgakis/navigation/recents/url/sharing/ProvideAssistContent.kt

And here is a test that passes that kinda showcases how it works when navigating to some deep-linkable destinations https://github.com/StylianosGakis/navigation-recents-url-sharing/blob/main/library/navigation-recents-url-sharing/src/androidTest/java/com/stylianosgakis/navigation/recents/url/sharing/ProvideAssistContentTest.kt